torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -1,44 +1,100 @@
 defmodule Torchx.NxDoctestTest do
+  @moduledoc """
+
+  # Why some doctests are excluded?
+
+  ## Not callbacks
+
+  * default_backend/2
+
+  ## Python rounds differently, than Elixir (for now)
+
+  * round/1
+
+  ## Result mismatch in less significant digits (precision issue)
+
+  * logistic/1
+
+  ## Callback or test requires explicity backend transfer to BinaryBackend
+
+  * to_binary/2
+  * to_flat_list/2
+  * random_uniform/4
+  * random_normal/4
+  * atan2/2
```

Comment:
This should work... no?

Reply:
Same story: tests use `Nx.to_binary/1`, which is banned.

```elixir
     iex> t = Nx.atan2(pos_and_neg_zero_x, pos_and_neg_zero_y)
      iex> Nx.to_binary(t)
      <<-3.141592653589793::float-32-native, (-1.0*(Integer.parse("0")|>elem(0)))::float-32-native,
        3.141592653589793::float-32-native, 0.0::float-32-native>>
```